src/mono/Directory.Build.props

```diff
@@ -12,7 +12,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <!-- Minimum target OS version, keep in sync with src/native/libs/build-native.sh -->
+    <!-- Minimum target OS version, keep in sync with src/native/libs/build-native.sh, eng/native/build-commons.sh -->
     <iOSVersionMin>10.0</iOSVersionMin>
```

Comment:
Similarly to what Michal suggested, these should be bumped to 11.0 to keep min versions in sync.